install/preview/prettylog/main.go

```diff
@@ -38,21 +39,31 @@ var (
 )
 
 func main() {
-	dmp, err := os.OpenFile("logs.txt", os.O_WRONLY|os.O_TRUNC|os.O_CREATE, 0644)
-	if err != nil {
-		panic(err)
+	// Warn and wait for user approval
+	pterm.FgLightCyan.Println(`
+Welcome to the local preview of Gitpod. Please note the following limitations:
+  - Performance is limited by the capabilities of your machine - a minimum of 4 cores and 6GB of RAM are required
+  - ARM CPUs including Macs with Apple Silicon (e.g. M1) are currently not supported
+For more information about these limitation, please visit the local preview documentation: https://www.gitpod.io/docs/self-hosted/latest/local-preview`)
+
+	result, _ := pterm.DefaultInteractiveConfirm.WithDefaultText("Continue?").WithDefaultValue(true).Show()
+	if !result {
+		// send telemetry for user exit
+		send_telemetry("user exit")
+		return
 	}
-	defer dmp.Close()
 
-	r := io.TeeReader(os.Stdin, dmp)
+	file, err := tail.TailFile("logs.txt", tail.Config{Follow: true})
```

Comment:
@MrSimonEmms Like we discussed in our call. This was because while `logs.txt` gets updated, We just read at a point and are exiting prettylog as there is EOF.

Instead we need a way to continuously read `logs.txt` as it gets updated, and this is where the `tail` package [comes in ](https://stackoverflow.com/a/15328871). By using that we are continuously reading from the latest updates, and thus updating status based on that.

Reply:
Was also able to test that this works as expected!